src/queryClient.ts

```diff
@@ -46,6 +47,14 @@ export default (config: Partial<QueryClientConfig>) => {
     staleTime: config?.staleTime || STALE_TIME_MILLISECONDS,
     // time before cache labeled as inactive to be garbage collected
     cacheTime: config?.cacheTime || CACHE_TIME_MILLISECONDS,
+    // derive WS_HOST from API_HOST if needed
+    // TODO: pass it with the context
+    WS_HOST:
+      config?.WS_HOST ||
+      process.env.VITE_WS_HOST ||
+      `${config?.API_HOST?.replace('http', 'ws')}/ws`,
+    // whether websocket support should be enabled
+    enableWebsocket: config?.enableWebsocket || false,
```

Comment:
Is this correct? May be more readable
```suggestion
    enableWebsocket: Boolean(config?.enableWebsocket),
```